packages/gitbook/src/middleware.ts

```diff
@@ -482,8 +482,8 @@ async function lookupSiteInMultiMode(request: NextRequest, url: URL): Promise<Lo
  *
  * The format of the path is:
  *   - /~space|~site/:id/:path
- *   - /~space|~site/:id/~changes/:changeId/:path
- *   - /~space|~site/:id/~revisions/:revisionId/:path
+ *   - /~space|~site/:id/~/changes/:changeId/:path
+ *   - /~space|~site/:id/~/revisions/:revisionId/:path
```

Comment:
Should we add backwards compatibility for old URLs to redirect to the proper one in case some users are already using the old format?

Reply:
No, there are not used anywhere (multi-id mode). Only the new preview was starting to use them.